### DISCUSSION TITLE
Change "request" url from ip address to domain url

### DISCUSSION THREAD

**Discussion by xlo0okix** [fastapi/fastapi#6348] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
import yaml

app = FastAPI()

def get_data():
    with open("/root/bot/price_data.yaml", "r") as d:
        data = yaml.load(d, Loader=yaml.FullLoader)
        return data

@app.get("/price_data")
async def price_data():
    data = get_data()
    return data

if __name__ == "__main__":
    uvicorn.run("api:app", host="0.0.0.0",  workers=3)
```

### Description

In /docs or /redoc when i execute a /get point, i get the response i want, but the "request url" is set to the ip address of my VPS instead of my domain url. i access /docs or /redoc over my domain.
 does someone have a solution for that?

*i am new to this stuff

### Operating System

Linux, Windows, macOS

### FastAPI Version

latest

### Python Version

latest

**Comment by YuriiMotov** [comment, marked as answer]
You can use `servers` parameter:

```python
from fastapi import FastAPI
import yaml

app = FastAPI(servers=[{"url": "http://myserver.com"}])

def get_data():
    with open("/root/bot/price_data.yaml", "r") as d:
        data = yaml.load(d, Loader=yaml.FullLoader)
        return data

@app.get("/price_data")
async def price_data():
    data = get_data()
    return data

```
